web-services/query/src/main/java/datawave/webservice/query/util/LookupUUIDUtil.java

```diff
@@ -773,49 +773,53 @@ public <T> T generateTagCloud(final AbstractUUIDLookupCriteria unvalidatedCriter
 
         BaseQueryResponse lastResponse = null;
 
-        do {
-            try {
-                // Get the first/next results
-                BaseQueryResponse response = this.queryExecutor.next(keywordQueryId);
-                lastResponse = response;
-
-                if (TagCloudResponseBase.class.isAssignableFrom(response.getClass())) {
-                    // merge tag cloud responses
-                    final TagCloudResponseBase tagCloudResponse = (TagCloudResponseBase) response;
-                    if (mergedResponse == null) {
-                        mergedResponse = tagCloudResponse;
+        try {
+            do {
+                try {
```

Comment:
try/do/try feels weird, but this mirrors what we do elsewhere in this class for consistency.